### ISSUE TITLE
[Bug]: shigatyper failed to open FASTA index

### ISSUE THREAD

**Issue by erinyoung** [StaPH-B/docker-builds#348]
### What container were you trying to use, and how were you attempting to use it?

I was running this with some of my samples (which is in the SRA as SRR7738178) and encountered the following fatal error. This does not happen with all of them.

### Relevant log output

```shell
Wed Apr 20 12:51:43 MDT 2022
[CRITICAL::ShigaTyper] ShigaTyper v2.0.0
[M::mm_idx_gen::0.005*1.26] collected minimizers
[M::mm_idx_gen::0.009*1.90] sorted minimizers
[M::main::0.011*1.72] loaded/built the index for 95 target sequence(s)
[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 95
[M::mm_idx_stat::0.011*1.68] distinct minimizers: 21938 (99.78% are singletons); average occurrences: 1.002; average spacing: 5.442; total length: 119664
[M::main] Version: 2.24-r1122
[M::main] CMD: minimap2 -d /tmp/tmpxif83es4/ShigellaRef5.mmi /miniconda/lib/python3.9/site-packages/shigatyper/resources/ShigellaRef5.fasta
[M::main] Real time: 0.013 sec; CPU: 0.021 sec; Peak RSS: 0.005 GB
[WARNING] Indexing parameters (-k, -w or -H) overridden by parameters used in the prebuilt index.
[M::main::0.005*1.11] loaded/built the index for 95 target sequence(s)
[M::mm_mapopt_update::0.005*1.11] mid_occ = 1000
[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 95
[M::mm_idx_stat::0.005*1.10] distinct minimizers: 21938 (99.78% are singletons); average occurrences: 1.002; average spacing: 5.442; total length: 119664
[M::worker_pipeline::1.406*2.00] mapped 217058 sequences
[M::worker_pipeline::2.197*2.31] mapped 216820 sequences
[M::worker_pipeline::3.009*2.46] mapped 217114 sequences
[M::worker_pipeline::3.759*2.62] mapped 216694 sequences
[M::worker_pipeline::3.917*2.60] mapped 106902 sequences
[M::main] Version: 2.24-r1122
[M::main] CMD: minimap2 -ax sr /tmp/tmpxif83es4/ShigellaRef5.mmi /Volumes/IDGenomics_NAS/testing_grandeur/2022-04-19/work/c0/a978611e491f2f8c420abb665e1b02/SRR7738178_clean_PE1.fastq.gz /Volumes/IDGenomics_NAS/testing_grandeur/2022-04-19/work/c0/a978611e491f2f8c420abb665e1b02/SRR7738178_clean_PE2.fastq.gz
[M::main] Real time: 3.919 sec; CPU: 10.179 sec; Peak RSS: 0.330 GB
[E::fai_build3_core] Failed to open FASTA index /miniconda/lib/python3.9/site-packages/shigatyper/resources/ShigellaRef5.fasta.fai : Read-only file system
[ERROR::ShigaTyper] Command 'samtools mpileup -C50 -q 20 -Q 20 -f /miniconda/lib/python3.9/site-packages/shigatyper/resources/ShigellaRef5.fasta /tmp/tmpxif83es4/_Shigella5.bam -o /tmp/tmpxif83es4/Shigella5.mpileup' returned non-zero exit status 1.
```

**Comment by kapsakcj** [comment]
I was able to reproduce this error. I think the docker image needs adjusting

**Comment by kapsakcj** [comment]
When I ran the container as the `root` user I was able to get shigatyper to run successfully. I would recommend opening up permissions rescursively, perhaps something pretty broad like `chmod -R a+rwx /miniconda/` oughta fix it

Could adjust this line: https://github.com/StaPH-B/docker-builds/blob/e92dd4d7a2e763138f9e8221cbeeafed88716fd9/shigatyper/2.0.1/Dockerfile#L93

CC @jwarnn

**Comment by erinyoung** [comment]
I don't run into permissions errors with https://github.com/StaPH-B/docker-builds/pull/351

**Comment by kapsakcj** [comment]
closed via #351
